### PR TITLE
4.2.3 bugfix 1026

### PR DESCRIPTION
**Pull request by Cilla-luodan**

### CODE REVIEW

File: markdown/RTC 4.x/migration-guide/en-US/migration_gude_ios_ng.md [hunk 1]
```diff
@@ -180,6 +180,20 @@ The behavior differences of Agora SDK between v3.7.0 and v4.0.0 are listed as fo
...
+
+- In v3.7.0, if you set `publishLocalAudio` in `AgoraRtcChannelMediaOptions` to `NO`, it will stop publishing the local audio stream within the channel.
+- In v4.0.0, if you set `publishMicrophoneTrack` in `AgoraRtcChannelMediaOptions` to `NO`, it will not only stop publishing the local audio stream within the channel but also stop local microphone capture.
+
+#### Video information change event
+
+If the video capture device is adjusted to landscape or portrait mode during video capture, in v3.7.0 and v4.0.0, there are the following differences in the video information change events reported by the SDK:
+
+- In v3.7.0, the `rotation` parameter in the `videoSizeChangedOfUid` callback displays the current device rotation information.
+- In v4.0.0, the `rotation` parameter in the `videoSizeChangedOfSourceType` callback is always `0`.
```
Comment (jinyuagora) [inline markdown/RTC 4.x/migration-guide/en-US/migration_gude_ios_ng.md:195]: is always -> is consistently set to

File: markdown/RTC 4.x/migration-guide/en-US/migration_gude_ios_ng.md [hunk 2]
```diff
@@ -180,6 +180,20 @@ The behavior differences of Agora SDK between v3.7.0 and v4.0.0 are listed as fo
...
+#### Media options
+
+There are differences in the behavior of the SDK when setting channel media options while joining a channel between v3.7.0 and v4.0.0:
+
+- In v3.7.0, if you set `publishLocalAudio` in `AgoraRtcChannelMediaOptions` to `NO`, it will stop publishing the local audio stream within the channel.
+- In v4.0.0, if you set `publishMicrophoneTrack` in `AgoraRtcChannelMediaOptions` to `NO`, it will not only stop publishing the local audio stream within the channel but also stop local microphone capture.
+
+#### Video information change event
+
+If the video capture device is adjusted to landscape or portrait mode during video capture, in v3.7.0 and v4.0.0, there are the following differences in the video information change events reported by the SDK:
```
Comment (jinyuagora) [inline markdown/RTC 4.x/migration-guide/en-US/migration_gude_ios_ng.md:192]: there are the following differences -> the following differences exist

File: dita/RTC-NG/config/keys-rtc-ng-api-ios.ditamap [hunk 3]
```diff
@@ -1288,7 +1288,7 @@
             <keydef keys="pushVideoFrameEx" href="../API/api_irtcengineex_pushvideoframeex.dita">
                 <topicmeta>
                     <keywords>
-                        <keyword>pushExternalVideoFrame</keyword>
```
Comment (jinyuagora) [inline dita/RTC-NG/config/keys-rtc-ng-api-ios.ditamap:1291]: mac 要改吗

File: markdown/RTC 4.x/migration-guide/en-US/migration_gude_ios_ng.md (hunk 1 above)
Comment (Cilla-luodan) [reply markdown/RTC 4.x/migration-guide/en-US/migration_gude_ios_ng.md:195]: 是在回调里上报的，不是人设置的。
所以不改啦

**Pull request merged**